docs/pages/mintlist/[mintlistAddress].tsx

```diff
@@ -471,49 +597,53 @@ function NftsGrid({ mintInfos }: { mintInfos: NftokenTypes.MintInfo[] }) {
   return (
     <>
       <div className="grid">
-        {mintInfosWithMetadata.map((mintInfo) => (
-          <figure className="nft-card" key={mintInfo.metadata_url}>
-            <img
-              className="nft-image"
-              alt={mintInfo.metadata.name}
-              src={mintInfo.metadata.image}
+        {mintInfosWithMetadata.map((mintInfo) => {
+          const nft = nftsData.get(mintInfo.metadata_url);
+
+          return nft ? (
+            <Link
+              href={`/nft/${nft.address}${
+                network !== Network.Mainnet ? `?network=${network}` : ""
```

Comment:
Let's use `qs.stringify` (`import qs from 'query-string'`) whenever doing query params since that's good practice. 

I believe you could also do `<LuxLInk href={} query={...}>`